docs/platforms/apple/guides/ios/session-replay/customredact.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Using Custom Redact for Session Replay
```

Comment:
Should we name it `Privacy` similar to the JS sdk? https://docs.sentry.io/platforms/javascript/session-replay/privacy

Reply:
I dont know, should we?

Reply:
I would also add here the `resume/pause` functionality, and I think in this case it'd make sense to rename it as it's not only custom redaction/masking anymore. Maybe `Privacy Controls` or something like that would do?

Reply:
Good point. I will improve in another doc. This is waiting for too long